CONTRIBUTING.md

```
@@ -1,77 +1,205 @@
+- [ ] Manual testing
+
+### Documentation
+- [ ] Updated docs
+- [ ] Added examples
+
+### Related Issues
+- Fixes #123
+- Related to #456
+```
+
+#### Review Process
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)